install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```
@@ -55,7 +55,7 @@ spec:
       rules:
         - alert: MCDRebootError
           expr: |
-             mcd_reboot_err > 0
+             increase(mcd_reboots_failed_total[15m]) > 0)
```

It's usually a good practice to keep only the labels that are relevant for the context. In this example, I assume that the namespace node and pod labels are enough and that other labels like  instance, service, ... don't bring any value.

```suggestion
             max by(namespace, node, pod) increase(mcd_reboots_failed_total[15m]) > 0
```